### PR TITLE
Test for Message Id Already Exist

### LINKED ISSUE #296: [C4GT] Network-Score: Add tests for 'MessageIdAlreadyExists'
## Description
Subtask under : https://github.com/cord-network/community/issues/7

New testcase can be tested under `cargo test -p pallet-network-score` after adding it in the code.

## Goals

- [ ] Add tests for `MessageIdAlreadyExists` for pallet/network-score

## Expected Outcome
- [ ] Test should assert for `MessageIdAlreadyExists` being returned properly in all the possible calls.
- [ ] If there are more than one function returning this error code, all of them should be validated either as one test case or as multiple test cases.
       
## Acceptance Criteria
NA

## Implementation Details
Look at other test cases, and add a test case for the same.
Ex: In pallets/network-membership check method `test_duplicate_member_request` which validates error code `MembershipAlreadyAcquired`

## Mockups / Wireframes
NA

---- 

## Product Name
CORD

## Organization Name
Dhiway

## Domain
Blockchain

## Tech Skills Needed
Rust

## Mentor(s)
@amarts 

## Complexity
[Low]

## Category
[Test]

## Sub Category
[Beginner friendly]

### PR DESCRIPTION
**Pull request by zeel991**
- Fixes https://github.com/dhiway/cord/issues/296
- @vatsa287 , I have removed all the changes and just pushed the test , please review the PR

### CODE REVIEW

**Comment by vatsa287** [comment]
@zeel991 Run `rustfmt pallets/network-score/src/tests.rs`.  Add changes and force push. 
Remove .gitignore file changes. It is not part of the fix.

File: .gitignore
Comment (zeel991) [inline .gitignore:1]: i have removed the changes

**Comment by zeel991** [comment]
Hey @vatsa287 , my cargo build test couldn't pass successfully due to an error `no space left on device` , I have passed all three tests , can you please look into this error and guide me if i have to do something in some other way .
Thank you .

**Comment by vatsa287** [comment]
> Hey @vatsa287 , my cargo build test couldn't pass successfully due to an error `no space left on device` , I have passed all three tests , can you please look into this error and guide me if i have to do something in some other way . Thank you .

Yeah it is infra-wide. Will re-trigger once it is solved.

**Comment by zeel991** [comment]
> > Hey @vatsa287 , my cargo build test couldn't pass successfully due to an error `no space left on device` , I have passed all three tests , can you please look into this error and guide me if i have to do something in some other way . Thank you .
> 
> Yeah it is infra-wide. Will re-trigger once it is solved.

Ohk @vatsa287 , Untill then can I start working on another issue . I ensure you that I wont make any unnecessary changes this time

**Comment by NiranjanAP** [comment]
@vatsa287  I noticed the test was successful.

image

**Comment by vatsa287** [comment]
> @vatsa287  I noticed the test was successful.
> 
> image
> 

Yes it ran successfully in my local setup as well. The issue is with GitHub actions which errors out saying lack of space. It is happening with all PRs on CORD.

**Comment by zeel991** [comment]
> > @vatsa287  I noticed the test was successful.
> > image
> 
> Yes it ran successfully in my local setup as well. The issue is with GitHub actions which errors out saying lack of space. It is happening with all PRs on CORD.

Ok , @vatsa287 So if my PR is correct , can you assign the issue to me and merge my PR or point out the error it has to be resolved .
Thank you .

**Comment by vatsa287** [comment]
@zeel991 Please rebase to latest. Steps below if required.
- Checkout to develop: `git checkout develop`
- Pull latest: `git pull upstream develop` & push update to you're fork: `git push origin develop`
- Checkout to you're feature branch: `git checkout youre_feature_branch`
- Rebase to latest: `git rebase origin/develop`.
- Push latest with `-f`.

**Comment by zeel991** [comment]
@vatsa287 I have tried and followed your steps .
Thank you so much for the instructions .

**Comment by zeel991** [comment]
Hey @vatsa287 , all the tests have passed successfully, please assign me the issue before merging my PR .
Thank you .

**Review by vatsa287 — APPROVED** [review]
Looks good to me.
@zeel991 since you have already completed a `good-first-issue`. You can focus on next level issues from [cord community](https://github.com/cord-network/community/issues).

**Comment by zeel991** [comment]
> Looks good to me. @zeel991 since you have already completed a `good-first-issue`. You can focus on next level issues from [cord community](https://github.com/cord-network/community/issues).

Waited for this message!!! , I'll start working on few basic problems and move on to next level problems .
Thank you so much @vatsa287 for patiently guiding me .

**Review by amarts — APPROVED** [review]

**Pull request merged**